flake.nix

```
@@ -198,25 +207,37 @@
 
         release-files = import ./nix/release-files.nix { inherit self pkgs; };
 
-        # Release version - excludes debug tests
+        # Common tests version - includes non-GC, non-release/debug specific tests.
+        common-systems-go = pkgs.releaseTools.aggregate {
+          name = "common-systems-go";
+          constituents = filter_tests "common";  # Only include common tests.
+        };
+
+        # GC tests version - only includes GC tests.
+        gc-systems-go = pkgs.releaseTools.aggregate {
```

What about using the following name which also matches its corresponding job:
```suggestion
        gc-tests = pkgs.releaseTools.aggregate {
```

---

thanks, fixed.